liota/dccs/iotcc.py

```diff
@@ -162,21 +162,20 @@ def on_response(msg):
                 log.debug("Processed msg: {0}".format(json_msg["type"]))
```

Comment:
Incorrect log, message is yet to be processed.

Reply:
Fixed.